app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/vm.rb

```diff
@@ -130,4 +130,58 @@ def sdk_update_status(instance)
       $ibm_cloud_log.info("VM instance #{instance.id} state is #{raw_power_state}")
     end
   end
+
+  def params_for_resize
```

Comment:
Does this need to be a public method?

Reply:
Yes, the API controller calls this from the object here https://github.com/ManageIQ/manageiq-api/blob/2a449e0e5e78595a03bcc165c80a896e88841b73/app/controllers/api/base_controller/renderer.rb#L557 not from an internal method